### PR TITLE
#9921 article page related articles

### PR DESCRIPTION
**Pull request by tbrlpld**
# Description

Because of some positioning issues outlined in #9664 with the primary related articles (the ones embedded in the content) we are moving to a simpler solution to define the related articles. This PR implements the first part of the simplification. 

This PR removes the primary/secondary concept for the related articles. 
We are now only using a single field and method for the related articles. 
The related articles are now displayed only in the bottom of the page (in the "what to read next" section). 
The JS that previously relocated the primary related links on the page is now removed. 
The number of related articles is reduced from 6 to 3, because only have one location to display the 3 related articles. 
This change is reflected in the factory too. 

Additionally, this PR makes some minor changes to the markup of the bottom section of the article page to achieve a more consistent implementation and spacing. 
It also removes the use of Bootstrap classes in that section and relies only on Tailwind classes. 

Link to sample test page: http://localhost:8000/en/privacynotincluded/articles/hair-pressure-tv-week-avoid-much/
Related PRs/issues: #9921

# Checklist

**Tests**
- [x] Is the code I'm adding covered by tests? Visual regression tests.

**Changes in Models:**
- [x] Did I update or add new fake data?
- ~~[ ] Did I squash my migration?~~ No migration added. 
- [x] Are my changes [backward-compatible](https://github.com/mozilla/foundation.mozilla.org/blob/main/docs/workflow.md#django-migrations-what-to-do-when-working-on-backward-incompatible-migrations). If not, did I schedule a deploy with the rest of the team?

**Documentation:**
- ~~[ ] Is my code documented?~~
- ~~[ ] Did I update the READMEs or wagtail documentation?~~

### CODE REVIEW

File: network-api/networkapi/templates/pages/buyersguide/article_page.html [hunk 1]
```diff
@@ -46,64 +46,51 @@ <h1 class="tw-h1-heading">{{ page.title }}</h1>
...
 
-  {% with related_articles=page.get_primary_related_articles  %}
-    {% if related_articles %}
-    <div class="tw-container tw-my-4">
-      <div class="tw-row">
-        <div id="article-primary-related-articles" class="tw-px-4 tw-w-8/12 tw-mx-auto">
-          {% include "fragments/buyersguide/related_reading.html" with articles=related_articles %}
+  {% with author_profiles=page.get_author_profiles  %}
+    {% if author_profiles %}
+      <div class="tw-container tw-mt-[3rem]">
```
Comment (danielfmiranda) [inline network-api/networkapi/templates/pages/buyersguide/article_page.html:54]:
```suggestion
      <div class="tw-container tw-mt-7">
```

File: network-api/networkapi/templates/pages/buyersguide/article_page.html [hunk 2]
```diff
@@ -46,64 +46,51 @@ <h1 class="tw-h1-heading">{{ page.title }}</h1>
...
       </div>
     </div>
   </div>
 
-  {% with related_articles=page.get_secondary_related_articles  %}
+  {% with related_articles=page.get_related_articles  %}
     {% if related_articles %}
       {% get_bg_home_page as bg_home_page %}
-      <div id="article-secondary-related-articles" class="tw-container">
+      <div class="tw-container tw-mt-[3rem]" data-gtm="article-what-to-read-next">
```
Comment (danielfmiranda) [inline network-api/networkapi/templates/pages/buyersguide/article_page.html:93]:
```suggestion
      <div class="tw-container tw-mt-7 data-gtm="article-what-to-read-next">
```

**Suggestion:** Same suggestion as above

File: network-api/networkapi/templates/pages/buyersguide/article_page.html (hunk 1 above)
Comment (danielfmiranda) [reply network-api/networkapi/templates/pages/buyersguide/article_page.html:54]: **Suggestion:** `tw-mt-7` equals 3rem and removes the need for a custom tw-class

**Review by danielfmiranda — APPROVED** [review]
Hi @tbrlpld thanks for taking this work on, after taking a look at the code and testing this out on my local machine, everything looks good to me and is working. Approved! 👍 

Also, I did leave two quick suggestions regarding the use of `tw-mt-[3rem]` if you want to take a look and consider using them

File: network-api/networkapi/templates/pages/buyersguide/article_page.html (hunk 1 above)
Comment (tbrlpld) [reply network-api/networkapi/templates/pages/buyersguide/article_page.html]: Great catch! Thanks @danielfmiranda

File: network-api/networkapi/templates/pages/buyersguide/article_page.html (hunk 2 above)
Comment (tbrlpld) [reply network-api/networkapi/templates/pages/buyersguide/article_page.html]: Great catch! Thanks @danielfmiranda

**Pull request merged**